src/deluge/hid/led/indicator_leds.h

```diff
@@ -22,30 +22,91 @@
 
 #define numLedBlinkers 4
 
+namespace IndicatorLEDs {
```

Comment:
as long as we're in the area, should probably use the `snake_case` namespace names (to match the menu item refactor)

Reply:
and apparently the `deluge::` root namespace too